### ISSUE TITLE
allow separation or padding between entities

### ISSUE THREAD

**Issue by gnkarn** [custom-cards/vertical-stack-in-card#16]
chart with std  vertical stack:
![screen shot 2018-08-06 at 10 07 31](https://user-images.githubusercontent.com/3314004/43718348-c517233a-9960-11e8-833a-7ca16c2f71cf.png)

Too much separation -

chart with custom stacking card

![screen shot 2018-08-06 at 10 06 54](https://user-images.githubusercontent.com/3314004/43718404-ee92ea14-9960-11e8-8d58-bec06a84f88a.png)

no separation separation .

of course i may be missing something , if there is a way to adjust it that im not aware of . thank you

**Comment by ofekashery** [comment]
Hi!
Are you using the latest version (0.0.5)?
Can you share your lovelace configuration?

**Comment by gnkarn** [comment]
code 

> 
` ############  system view   ############   `  
 ```
 - icon: mdi:laptop
    title: Sistema
    # Each view can have a different theme applied. Theme should be defined in the frontend.
    theme: dark-mode
    # The cards to show on this view.
```
    cards:
      - type: horizontal-stack
        cards:
        - type: vertical-stack
          cards:
          - type: custom:bignumber-card
            title: Disk free
            entity: sensor.disk_free_home
            scale: '20px'
            from: left
            min: 0
            max: 32
            severity:
              - value: 10
                style: 'var(--label-badge-red)'
              - value: 20
                style: 'var(--label-badge-yellow)'
              - value: 32
                style: 'var(--label-badge-green)'
          - type: custom:bignumber-card
            title: cpu temp
            entity: sensor.cpu_temperature
            scale: '20px'
            from: left
            min: 0
            max: 80
            severity:
              - value: 45
                style: 'var(--label-badge-green)'   
              - value: 60
                style: 'var(--label-badge-yellow)'
              - value: 80
                style: 'var(--label-badge-red)'

**Comment by ofekashery** [comment]
Are you using the latest version (0.0.5)?

**Comment by ofekashery** [comment]
Fixed in new version (0.0.6).

**Comment by gnkarn** [comment]
tested 006 , it is now better aligned , but no way to separate  one measure from the other , 
i think what is needed is to add a CSS padding option to styling , in order to make it useful . tks
